ansible/run.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: local
+  connection: local
+  vars_files:
+      - runtime_vars.yml
```

Comment:
There's a need to disambiguate between `build_vars` (used when building the container with Ansible by build.yml, but not used in this particular container) and `run_vars` (used when running the containers, used by run.yml). I've renamed runtime_vars to run_vars to be more consistent.